test/unit/CMakeLists.txt

```diff
@@ -28,8 +28,10 @@ ADD_SUBDIRECTORY("core/")
 ADD_SUBDIRECTORY("event/")
 ADD_SUBDIRECTORY("idx/")
 ADD_SUBDIRECTORY("kernel/")
-ADD_SUBDIRECTORY("math/sincos/")
 ADD_SUBDIRECTORY("mem/buf/")
+ADD_SUBDIRECTORY("math/sincos/")
```

Comment:
It is probably a little more logical to put these 3 lines before the `"mem/..."` includes